dashmips/instructions/imm_instructions.py

```diff
@@ -15,7 +16,10 @@ def parse(args: Tuple[str, str, str]) -> Tuple[str]:
 @mips_instruction(PATTERN, parse, label=True)
 def j(program: MipsProgram, address: str):
     """Jump unconditionally to label."""
-    program.registers["pc"] = program.labels[address].value - 1
+    if address in program.labels:
+        program.registers["pc"] = program.labels[address].value - 1
+    else:
+        raise MipsException('"j": too many or incorrectly formatted operands')
```

Comment:
Shouldn't the regex catch the "too many" case? maybe its just non-existent label error